src/main/java/com/github/dockerjava/api/model/Image.java

```diff
@@ -31,12 +32,24 @@
     @JsonProperty("RepoTags")
     private String[] repoTags;
 
+    @JsonProperty("RepoDigests")
+    private String[] repoDigests;
```

Comment:
No arrays https://github.com/docker-java/docker-java/blob/master/docs/devel.adoc

Reply:
This field just like "RepoTags" , so it defined to arrays , if it's incorrect please let me know the detail thanks

Reply:
we are moving from arrays to List

Reply:
Could you change it?

Reply:
@KostyaSha Please check it